web/src/icons/emapsIcon.tsx

```diff
@@ -1,9 +1,9 @@
-export function EmapsIcon() {
+export function EmapsIcon({ size = 22, styling }: { size?: number; styling?: string }) {
   return (
-    <div className="text-black dark:text-white">
+    <div className={`text-black ${styling || ''}`}>
```

Comment:
In other places we have used className (to extend the native class name usage) and then tailwind merge to ensure they are not conflicting.

Could we do the same here?